Firestore/Source/Local/FSTLocalWriteResult.mm

```diff
@@ -29,11 +33,15 @@ - (instancetype)initWithBatchID:(BatchId)batchID
 @end
 
 @implementation FSTLocalWriteResult {
-  MaybeDocumentMap _changes;
+  DelayedConstructor<LocalWriteResult> _write_result;
```

Comment:
Also, feel free to make `LocalWriteResult` default constructible so that you can eliminate this `DelayedConstructor` usage.

Reply:
Deleted file.